feat<client>: ui overhaul part 3

dip-land opened the pull request:
Improved user mentions
Overhauled server list ui
Overhauled channel list ui
Overhauled chat ui
Fixed messageField allowing images and rich text

dip-land committed:
fix<client>: remove message empty popup

dip-land committed:
feat<client>: UI user list

User List UI overhauled
Fixed some mobile bugs

dip-land committed:
feat<client>: UI user popup & message rework

Added user popups
Reworked how messages are displayed

dip-land committed:
Delete undefined

dip-land committed:
fix<client>:  context menu

Removed default context menu from entire page
Removed unused css
Caught up to commit 7e012f9

dip-land committed:
feat<client>: UI user popup styling

Styled user popups
Fixed user popup bugs
Added user popup hide on window resize

dip-land committed:
Merge branch 'uwu' into dip-dev

dip-land committed:
Update README.md

dip-land committed:
Merge branch 'TenDRILLL:uwu' into dip-dev

dip-land committed:
feat<client>: ui overhaul part 3

dip-land committed:
Merge branch 'dev' into dip-dev

TenDRILLL commented:
src/html/pages/app.ejs has static urls pointing to headpat prod, we should make them dynamic to support deployments in other envs, example here https://github.com/TenDRILLL/Headpat/pull/11/files#diff-d06436c22624fc554c897215b838a3e3df84e227353fab2b8ad4a64a8245e137R30

I know this mistake was also done in the original code.

dip-land committed:
fix<client>: some bugs

Made some static images into dynamic images
Fixed a small mention styling mistake

dip-land committed:
Merge branch 'dip-dev' of https://github.com/dip-land/Headpat into dip-dev

dip-land commented:
> src/html/pages/app.ejs has static urls pointing to headpat prod, we should make them dynamic to support deployments in other envs, example here https://github.com/TenDRILLL/Headpat/pull/11/files#diff-d06436c22624fc554c897215b838a3e3df84e227353fab2b8ad4a64a8245e137R30
> 
> I know this mistake was also done in the original code.

I have fixed that with a new commit
